packages/component-library-react/src/Calendar/CalendarNavigationButtons.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+interface CalendarNavigationButtonsProps
+  extends React.DetailedHTMLProps<React.ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement> {
+  onPreviousClick?: React.MouseEventHandler<HTMLButtonElement>;
+  onNextClick?: React.MouseEventHandler<HTMLButtonElement>;
+  previousIcon?: any;
+  nextIcon?: any;
+  previousTitle?: string;
+  nextTitle?: string;
+  children?: React.ReactNode;
+}
+
+export const CalendarNavigationButtons: React.FC<CalendarNavigationButtonsProps> = ({
+  onPreviousClick,
+  onNextClick,
+  previousIcon,
+  nextIcon,
+  children,
+  ...props
+}) => (
+  <div className="utrecht-calendar__navigation-buttons">
+    <button className="utrecht-calendar__navigation-buttons-item" onClick={onPreviousClick} {...props}>
+      <div className="utrecht-calendar__navigation-buttons-item-icon">{previousIcon}</div>
+    </button>
```

Comment:
I would love you to try using `<Button appearance="subtle-button">` instead of a vanilla `<button>` and configuring it the design tokens:
```css
.utrecht-calendar__navigation-buttons {
  --utrecht-button-subtle-color: currentColor;
}
```
or something along those lines